jobs/garden/templates/garden_ctl.erb

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash -e
 
+<% if_p('env.http_proxy') do |http_proxy| %>
+	export HTTP_PROXY=<%= http_proxy %>
+	export http_proxy=<%= http_proxy %>
+<% end %>
+<% if_p('env.https_proxy') do |https_proxy| %>
+	export HTTPS_PROXY=<%= https_proxy %>
+	export https_proxy=<%= https_proxy %>
+<% end %>
+<% if_p('env.no_proxy') do |no_proxy| %>
+	export NO_PROXY=<%= no_proxy %>
+	export no_proxy=<%= no_proxy %>
+<% end %>
```

Comment:
Since the goal is to have the garden-linux code that pulls docker images respect the proxy configured in these environment variables, it probably makes sense to pick the versions that Golang respects. From http://golang.org/pkg/net/http/#ProxyFromEnvironment, these are the `ALL_CAPS` versions, or the `no_caps` versions if those are not set. Also worth checking that the client's RoundTripper is an `http.Transport` with `http.ProxyFromEnvironment` as its Proxy function. If it's the DefaultTransport, it should be.

Generally, it seems that in some cases the all-uppercase versions are respected, and in others the all-lowercase versions are:
- http://serverfault.com/questions/571887/why-wget-doesnt-read-http-proxy-in-uppercase-format
- http://curl.haxx.se/docs/manpage.html#ENVIRONMENT

May be best just to set both styles to be thorough?